extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -547,9 +549,12 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 		if (component.value === originalValue) {
 			return false;
 		} else if ((!component.valid)) {
+			this.discardButton.enabled = true;
```

Comment:
This is being called every time the text changes - which means that it could steal focus away in the middle of typing something or changing values. Is that not happening or is that the behavior you want?

It's unclear to me what the person is actually saying is wrong. Do you have better context on what the issue is there? Otherwise I'd ask for clarification because I'm not sure what they're saying should happen here. Where does the focus go currently?

Reply:
Your point makes sense to me. I will ask for clarification. Currently the focus does not go anywhere after an input box has been edited.

Reply:
Removed from this PR